ide-common/src/main/java/org/digma/intellij/plugin/settings/SettingsComponent.java

```diff
@@ -106,15 +107,26 @@ public boolean verify(JComponent input) {
 
 
         myEmbeddedJaegerMessage.setForeground(JBColor.BLUE);
+
         var myJaegerLinkModeLabel = new JBLabel("Jaeger link mode: ");
-        myJaegerLinkModeLabel.setToolTipText("Internal will open the link as an embedded URL within the IDE. " +
-                "External will open the link externally to your default browser." +
-                " Embedded mode will open embedded Jaeger UI in the editor area.");
+        myJaegerLinkModeLabel.setToolTipText(""
+                + "Internal will open the link as an embedded URL within the IDE. "
+                + "External will open the link externally to your default browser. "
+                + "Embedded mode will open embedded Jaeger UI in the editor area. "
+        );
         myJaegerLinkModeComboBox.addItemListener(e -> {
             LinkMode selected = (LinkMode) myJaegerLinkModeComboBox.getSelectedItem();
             linkModeSelected(defaultLabelForeground, selected);
         });
 
+        var mySpringBootObservabilityModeLabel = new JBLabel("Spring Boot observability mode: ");
+        mySpringBootObservabilityModeLabel.setToolTipText(""
+                + "OtelAgent will use the default OpenTelemetry java agent. "
+                + "Micrometer will use Micrometer tracing, including the annotation of 'Observed' "
+        );
+        mySpringBootObservabilityModeComboBox.addItemListener(e -> {
+            SpringBootObservabilityMode selected = (SpringBootObservabilityMode) mySpringBootObservabilityModeComboBox.getSelectedItem();
+        });
```

Comment:
What does it do?

Reply:
actually nothing... to many copy-paste :-) 
i'll remove it